src/containers/HandleItemInfo/HandleItemInfo.jsx

```diff
@@ -61,58 +69,83 @@ function HandleItemInfo({
     changeActivityFlag();
     setCollectionItem({ ...collectionItem, description: value });
   }
-  function changeCollection(value) {
+  async function addNewCollection(value) {
+    await dispatch(addCollectionToAPI(value)).then((data) => {
+      setSelectedOptionValue(data.payload);
+      setCollectionItem({
+        ...collectionItem,
+        collection: Number(data.payload.id),
+      });
+    });
     changeActivityFlag();
-    setCollectionItem({ ...collectionItem, collection: value.label });
+  }
+
+  function changeExistedCollections(value) {
     setSelectedOptionValue(value);
+    setCollectionItem({
+      ...collectionItem,
+      collection: Number(value.id),
+    });
   }
+
   function changeItemPrice(value) {
     changeActivityFlag();
     setCollectionItem({ ...collectionItem, price: value });
   }
 
   return (
     <>
-      <PageTitle>{pageTitle}</PageTitle>
-      <Flex>
-        <Flex direction={"column"} justify={"center"} align="center">
-          <Card edited={true} {...collectionItem} />
-        </Flex>
-        <Flex direction={"column"} justify={"center"}>
-          <ComponentsWrapper mb={"10"}>
-            <Select
-              onKeyDown={changeActivityFlag}
-              selectedItem={itemCollection}
-              options={collections}
-              select={changeCollection}
-            />
-          </ComponentsWrapper>
-          <Input
-            ref={inputTitle}
-            value={collectionItem.title ? collectionItem.title : ""}
-            onChange={addTitle}
-            placeholder="Add title..."
-          />
-          <ComponentsWrapper mb={"5"} mt={"10"}>
-            <PriceInput
-              value={collectionItem.price ? collectionItem.price : ""}
-              onChange={changeItemPrice}
-            ></PriceInput>
-          </ComponentsWrapper>
-          <ComponentsWrapper mb={"5"} mt={"10"}>
-            <TextArea
-              value={collectionItem.description}
-              changeDescription={changeNewItemDescription}
-            />
-          </ComponentsWrapper>
-          <Button
-            onClick={() => onSubmit(collectionItem, selectedOptionValue)}
-            warning
-          >
-            {buttonText}
-          </Button>
-        </Flex>
-      </Flex>
+      {collections.length > 0 ? (
```

Comment:
из-за того что компонента довольно большая становится необходимо проскролить до конца, чтобы понять что у нас должно отобразиться если нет элементов. эта компонента позволяет нам разделить ретурн на два


```suggestion
      if(!collections.length) {
          return <Loader />
      }
      
      return ....
```